ngraph/src/ngraph/runtime/reference/ctc_loss.hpp

```diff
@@ -0,0 +1,137 @@
+// Copyright (C) 2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include "ngraph/shape_util.hpp"
+
+namespace ngraph
+{
+namespace runtime
+{
+namespace reference
+{
+
+template <typename T, typename U>
+void CTCLoss(const T* logits,
```

Comment:
Can you add tests for this implementation?

Reply:
It's supposed, that plugins will test that. CPU for example https://github.com/openvinotoolkit/openvino/pull/1482